CMR/python/demos/scripts/vars.py

```diff
@@ -17,16 +17,16 @@ def get_block_of_records(search, env=None):
     # build a url
     if env is None:
```

Comment:
Just a thought - you are doing this setting the correct url based on env in a few places. Maybe abstract it into a method in common?